commands/create-registry.ts

```diff
@@ -0,0 +1,137 @@
+
+import * as vscode from "vscode";
+import { ContainerRegistryManagementClient } from 'azure-arm-containerregistry';
+import { AzureAccountWrapper } from '.././explorer/deploy/azureAccountWrapper';
+import { SubscriptionClient, ResourceManagementClient, SubscriptionModels } from 'azure-arm-resource';
+import { AzureAccount, AzureSession } from '../typings/azure-account.api';
+import { RegistryRootNode } from "../explorer/models/registryRootNode";
+import { ServiceClientCredentials } from 'ms-rest';
+import { RegistryNameStatus } from "azure-arm-containerregistry/lib/models";
+const teleCmdId: string = 'vscode-docker.createRegistry';
+import { ResourceGroup, ResourceGroupListResult } from "azure-arm-resource/lib/resource/models";
+import { AzureCredentialsManager } from '../utils/AzureCredentialsManager';
+
+
+export async function createRegistry() {
```

Comment:
Note, we should check for being logged in to azure first but this has to be done once the Unified client PR is done